### PR TITLE
Add Border to Source Detail Error Message

### PR DESCRIPTION
**Pull request by joshri**
Closes: #1621

Changed structure of the header area of `SourceDetail` to clean things up and get a spicy border around that error message

image

### CODE REVIEW

File: ui/components/SourceDetail.tsx [hunk 1]
```diff
@@ -23,6 +23,24 @@ type Props = {
   info: <T>(s: T) => InfoField[];
 };
 
+type StatusProps = {
+  status?: string;
+  msg: string;
+};
+const PageStatus = ({ status, msg }: StatusProps) => {
```
Comment (jpellizzari) [inline ui/components/SourceDetail.tsx:30]: This needs to be repeated on automations as well right?

Comment (joshri) [reply ui/components/SourceDetail.tsx:30]: Yeah it needs to be done on everything that doesn't have source table - I made a new issue for it but I should just separate this out here.

File: ui/components/PageStatus.tsx [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import Text from "./Text";
+
+type StatusProps = {
+  ok: boolean;
+  msg: string;
+  error: boolean;
+  className?: string;
+};
+
+function PageStatus({ ok, msg, error, className }: StatusProps) {
```
Comment (jpellizzari) [inline ui/components/PageStatus.tsx:15]: What if this accepted a `Conditions[]` type?

**Review by jpellizzari — APPROVED** [review]
LGTM, only nits

File: ui/components/PageStatus.tsx [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+  console.log(error);
```
Comment (jpellizzari) [inline ui/components/PageStatus.tsx:16]: Extra `console.log` here.

**Review by jpellizzari — APPROVED** [review]

**Pull request merged**